libs/blocks/merch/merch.js

```diff
@@ -521,13 +521,21 @@ export async function initService(force = false, attributes = {}) {
     fetchEntitlements.promise = undefined;
     fetchCheckoutLinkConfigs.promise = undefined;
   }
-  const { commerce, locale: miloLocale } = getConfig();
+  const { commerce, env, locale: miloLocale } = getConfig();
   initService.promise = initService.promise ?? polyfills().then(async () => {
     await import('../../deps/mas/commerce.js');
     const { language, locale } = getMiloLocaleSettings(miloLocale);
     let service = document.head.querySelector('mas-commerce-service');
     if (!service) {
-      service = createTag('mas-commerce-service', { env: commerce?.env?.name, locale, language, ...attributes, ...commerce });
+      service = createTag('mas-commerce-service', {
+        locale,
+        language,
+        ...attributes,
+        ...commerce,
+      });
+      if (env?.name !== 'prod') {
+        service.setAttribute('allow-override', '');
```

Comment:
just a suggestion, to define all attributes in one go we can move allowOverride above:
```

const allowoverrides = env?.name !== 'prod';
service = createTag('mas-commerce-service', {
        locale,
        language,
        allowoverrides,
        ...attributes,
        ...commerce,
      });
```

but yeah, then the attr name is not great.
so up to you :)

Reply:
I don't want to set `allow-override` if it is `false` which is the case on production.